task-sdk/src/airflow/sdk/execution_time/task_runner.py

```diff
@@ -33,6 +34,7 @@
 
 import aiologic
 import attrs
+import jinja2
```

Comment:
Please don't import this at the top level - it can be a surprisingly expensive module to import